build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'scala'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
```

Comment:
Newer Kotlin versions would require updating Gradle

Reply:
`'1.4.30'` (latest) I wonder how large is this difference actually, is it like Scala's 2.12 vs 2.13 or else?

Reply:
It doesn't look like a big difference. They seem to be adding new versions very often (1.3.72 is still less than a year old).

Reply:
Should we update?

Reply:
But anyways that can be done separately from this PR